.gitignore

```diff
@@ -137,6 +137,7 @@ JLinkLog.txt
 
 # build directory
 build/
+cmake-build-debug/
```

Comment:
Also add `cmake-build-release`

Reply:
Do you mean that I should remove `cmake-build-debug`? This is the default CLion building directory, so I think we should add it just in case.

Reply:
I mean add another one, for release build

Reply:
Cool